backend/backEnd_constants.py

```diff
@@ -14,29 +15,36 @@ def __init__(self, cavityNum, rackObject):
         self.statusPV = PV(self.pvPrefix + STATUS_SUFFIX)
         self.severityPV = PV(self.pvPrefix + SEVERITY_SUFFIX)
 
-        self.faultPVs = []
-        for fault in faults:
-            # Decided to use timeout of 0.01 seconds after some trial and error
-            timeout = 0.01
-            if fault.level == "RACK":
-                if fault.rack != self.rack.rackName:
+        self.faults: List[Fault] = []
+
+        for csvFault in CSV_FAULTS:
+
+            if csvFault["Level"] == "RACK":
+
+                # Rack A cavities don't care about faults for Rack B and vice versa
+                if csvFault["Rack"] != self.rack.rackName:
                     continue
```

Comment:
Python question: Does "continue" take us out of the current if statement, the above if statement, or does it just take us out of the current iteration in the for loop?

Reply:
It takes us to the next iteration of the for loop! I can add a comment